mlos_bench/mlos_bench/config/services/remote/azure/scripts/setup-rg/results-db/mysql-template.json

```diff
@@ -0,0 +1,94 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "dbName": {
+            "type": "string"
+        },
+        "dbLocation": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]"
+        },
+        "dbAdminUsername": {
+            "type": "string",
+            "defaultValue": "mlos"
+        },
+        "dbAdminPassword": {
+            "type": "securestring"
+        },
+        "dbFirewallRules": {
+            "type": "array",
+            "defaultValue": []
+        }
+    },
+    "functions": [],
+    "variables": {},
+    "resources": [
+    {
+        "type": "Microsoft.DBforMySQL/flexibleServers",
+        "apiVersion": "2021-12-01-preview",
+        "name": "[parameters('dbName')]",
+        "location": "[parameters('dbLocation')]",
+        "sku": {
+            "name": "Standard_B1s",
+            "tier": "Burstable"
+        },
+        "properties": {
+            "administratorLogin": "[parameters('dbAdminUsername')]",
+            "administratorLoginPassword": "[parameters('dbAdminPassword')]",
+            "storage": {
+                "storageSizeGB": 20,
+                "iops": 360,
+                "autoGrow": "Enabled",
+                "autoIoScaling": "Disabled",
+                "logOnDisk": "Disabled"
```

Comment:
Can we leave those two settings unspecified to let whatever upstream defaults are take effect?
That way it's less for us to maintain and users hopefully get whateever enhancements happen on the upstream service for free.

Reply:
Removed these settings to be unspecified.